### PR TITLE
[Android] Adding support for big image notifications

### PR DESCRIPTION
**Pull request by shivam-aditya**
I have made changes to support big image notifications for the library. Also made changes to the readme reflecting the same.

### CODE REVIEW

**Comment by shivam-aditya** [comment]
This is exactly same as Pull request #100 with just some changes made with respect to the updated status of the library.

File: android/src/main/java/com/dieam/reactnativepushnotification/modules/RNPushNotificationHelper.java [hunk 1]
```diff
@@ -21,6 +21,20 @@
 import android.support.v4.app.NotificationCompat;
 import android.util.Log;
 
+import android.support.annotation.Nullable;
```
Comment (chenop) [inline android/src/main/java/com/dieam/reactnativepushnotification/modules/RNPushNotificationHelper.java:24]: Suggest to use the androidx implementation

import androidx.annotation.NonNull;

File: android/src/main/java/com/dieam/reactnativepushnotification/modules/RNPushNotificationHelper.java [hunk 2]
```diff
@@ -348,6 +375,41 @@ public void sendToNotificationCentre(Bundle bundle) {
...
+                .newBuilderWithSource(Uri.parse(imageUrl))
+                .setRequestPriority(Priority.HIGH)
+                .setLowestPermittedRequestLevel(ImageRequest.RequestLevel.FULL_FETCH)
+                .build();
+        DataSource<CloseableReference<CloseableImage>> dataSource =
+                imagePipeline.fetchDecodedImage(imageRequest, context);
+
+        dataSource.subscribe(new BaseBitmapDataSubscriber() {
+            @Override
+            public void onNewResultImpl(@Nullable Bitmap bitmap) {
```
Comment (chenop) [inline android/src/main/java/com/dieam/reactnativepushnotification/modules/RNPushNotificationHelper.java:398]: public void onNewResultImpl(@NonNull Bitmap bitmap) {

**Comment by chenop** [comment]
Who can approve this? 
Who is in charge now?

**Comment by chenop** [comment]
@zo0r  Can you please merge this issue?

**Comment by Dallas62** [comment]
Hi,
A version is available on `dev`.
Thanks for your contribution !

**Pull request closed without merging**